### PR TITLE
[Windows] Clean apps windows projects

### PR DESCRIPTION
**Pull request by talregev**
Clean apps windows projects.

### CODE REVIEW

File: win_build/htmlgfx_vs2019.vcxproj
Comment (AenBleidd) [inline win_build/htmlgfx_vs2019.vcxproj:98]: This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

File: win_build/vboxwrapper_vs2019.vcxproj
Comment (AenBleidd) [inline win_build/vboxwrapper_vs2019.vcxproj:98]: This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

File: win_build/worker_vs2019.vcxproj
Comment (AenBleidd) [inline win_build/worker_vs2019.vcxproj:96]: This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

File: win_build/wrapper_vs2019.vcxproj
Comment (AenBleidd) [inline win_build/wrapper_vs2019.vcxproj:93]: This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: win_build/htmlgfx_vs2019.vcxproj
Comment (talregev) [reply win_build/htmlgfx_vs2019.vcxproj:98]: The architecture is x64.

File: win_build/vboxwrapper_vs2019.vcxproj
Comment (talregev) [reply win_build/vboxwrapper_vs2019.vcxproj:98]: The architecture is x64.

File: win_build/worker_vs2019.vcxproj
Comment (talregev) [reply win_build/worker_vs2019.vcxproj:96]: The architecture is x64.

File: win_build/wrapper_vs2019.vcxproj
Comment (talregev) [reply win_build/wrapper_vs2019.vcxproj:93]: The architecture is x64.

File: win_build/htmlgfx_vs2019.vcxproj
Comment (AenBleidd) [reply win_build/htmlgfx_vs2019.vcxproj:98]: And should be `x86_64`

File: win_build/vboxwrapper_vs2019.vcxproj
Comment (AenBleidd) [reply win_build/vboxwrapper_vs2019.vcxproj:98]: And should be `x86_64`

File: win_build/wrapper_vs2019.vcxproj
Comment (AenBleidd) [reply win_build/wrapper_vs2019.vcxproj:93]: And should be `x86_64`

File: win_build/worker_vs2019.vcxproj
Comment (AenBleidd) [reply win_build/worker_vs2019.vcxproj:96]: And should be `x86_64`

**Comment by AenBleidd** [comment]
image

File: win_build/vboxwrapper_vs2019.vcxproj
Comment (talregev) [reply win_build/vboxwrapper_vs2019.vcxproj:98]: And should be `x64`

File: win_build/worker_vs2019.vcxproj
Comment (talregev) [reply win_build/worker_vs2019.vcxproj:96]: And should be `x64`

File: win_build/wrapper_vs2019.vcxproj
Comment (talregev) [reply win_build/wrapper_vs2019.vcxproj:93]: And should be `x64`

File: win_build/htmlgfx_vs2019.vcxproj
Comment (talregev) [reply win_build/htmlgfx_vs2019.vcxproj:98]: And should be `x64`

**Comment by AenBleidd** [comment]
@talregev, if you don't want to collaborate - I see no reason to continue useless discussion

**Comment by talregev** [comment]
When you give a real explanation for the naming, and not copy part, with strong reasoning, We can start collaborate.
I agree your arguments was useless.

**Comment by AenBleidd** [comment]
> When you give a real explanation for the naming, and not copy past, with strong reasoning, We can start collaborate. I agree your arguments was useless.

I wrote to you:

> This will change architecture name from `x86_64` to `x64`. For compatibility target name architecture should be `x86_64`.

What is not clear with the words `for compatibility`?

**Comment by talregev** [comment]
It not strong reason. and you need to explain more if you want to preserve this compatibility

**Comment by AenBleidd** [comment]
Why I should spend time to explain you what does `for compatibility` mean?
We have such a naming scheme for years (for example, you could see it here: https://boinc.berkeley.edu/dl/), and this is exactly `you` who need a strong reason to break this scheme now.
If I told you that smth should be made some way - that means that there is a reason for this, not just my wish. And I don't need to explain every time why.
I don't want to waste my time arguing for nothing.

**Comment by talregev** [comment]
Don't waste your time, I just create another PR.

**Comment by talregev** [comment]
I create this change mainly because we have a bug in our rename process:
worker of arm64 is also the same name of worker of x64. 

https://github.com/BOINC/boinc/pull/4678/files#diff-971d8b10f183ac019a1e5d61cbe4673ee113dddd77ed3e1441cc25b2fddfb343L105

You try to explain your logic but I am still not agree to it:
> I'll repeat one more time: some of the binaries are expected to have the exact architecture 'x86_64' because historically we have exactly it. And to not break plenty of projects that uses our binaries, we must not just rename it because you think it would be more correct.

Because we change the version number of all the binaries you complain with the architecture name. It mean the maintainers need to adapt the new binary with the new number. Also you remove versioning from some of the binaries, so the maintainers should to the new names as well.

So my strong motivation is to prevent this bug I show up, by generalize the process name, and make them more accurate. 
So in my precpective, you save some of backward compatibility in name, and some not. And if we already made a change, let fix the bug and have more new and accurate and automated binaries name.

**Comment by talregev** [comment]
Please remove the wafting for author respond, because I already respond.

**Comment by AenBleidd** [comment]
Since the main question is not resolved, I can't merge it. I keep it open for further discussion

**Review by AenBleidd — CHANGES_REQUESTED** [review]
Requires further discussion

**Pull request closed without merging**
